### DISCUSSION TITLE
Format Editor

### DISCUSSION THREAD

**Discussion by lamprey13** [mtxr/vscode-sqltools#1032] [Q&A]
Is there a Format configuration editor or something that allows you to configure how SQLTools will format SQL?

Stuff like newlines before/after commas and such?

If there is I can't seem to find it, so I wanted to verify.

Thanks!

**Comment by Norlandz** [comment]
Im not sure.
But if the following doc indeed shows **all** the settings,
Then it seems only very **few** options are provided.

> Find out more in the documentation [here](http://vscode-sqltools.mteixeira.dev/#features).
> \>
> https://vscode-sqltools.mteixeira.dev/en/settings & https://vscode-sqltools.mteixeira.dev/en/playground/formatter
> \>
> sqltools.format
